### DISCUSSION TITLE
Add support for S3 caching

### DISCUSSION THREAD

**Discussion by tamirkiviti13** [aquasecurity/trivy#8155] [Ideas]
### Description

Today trivy supports multiple options for scan caching - https://trivy.dev/latest/docs/configuration/cache.
Adding a new option to integrate an S3 bucket for caching might be useful.

### Target

None

### Scanner

None

**Comment by simar7** [comment]
We've discussed this in the past - see here for details https://github.com/aquasecurity/trivy/pull/6732#issuecomment-2209907787

**Reply by knqyf263** [reply]
@itaysk Just for the heads-up.
